frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -73,19 +73,7 @@ export class RhvImportWizard extends ImportWizard {
     );
   }
 
-  async import(config: VMImportConfig) {
-    const {
-      provider,
-      instanceConfig,
-      name,
-      description,
-      sourceVMName,
-      storageResources,
-      networkResources,
-      startOnCreation,
-    } = config;
-    await this.openWizard(VirtualMachineModel);
-
+  async importInitStep(provider: string, instanceConfig: InstanceConfig, sourceVMName: string) {
```

Comment:
can we name it accordingly? eg `connectToProviderStep` here?

Reply:
Done